surefire-api/src/test/java/org/apache/maven/surefire/api/util/RunOrderCalculatorTest.java

```diff
@@ -59,4 +63,65 @@ public void testOrderTestClasses()
     {
 
     }
+
+    public void testOrderTestMethodsNonRegex()
```

Comment:
Move all of these test methods into ./surefire-api/src/test/java/org/apache/maven/surefire/api/testset/TestListResolverTest.java